js/runtime.ts

```diff
@@ -344,16 +347,20 @@ class TypeScriptHost implements ts.LanguageServiceHost {
     return moduleNames.map((name: string) => {
       let resolvedFileName;
       if (name === "deno") {
-        resolvedFileName = resolveModuleName("deno.d.ts", "/$asset$/");
+        resolvedFileName = resolveModuleName("deno.d.ts", ASSETS);
       } else if (name === "typescript") {
-        resolvedFileName = resolveModuleName("typescript.d.ts", "/$asset$/");
+        resolvedFileName = resolveModuleName("typescript.d.ts", ASSETS);
       } else {
         resolvedFileName = resolveModuleName(name, containingFile);
         if (resolvedFileName == null) {
           return undefined;
         }
       }
-      const isExternalLibraryImport = false;
+      // This flags to the compiler to not go looking to transpile functional
+      // code, anything that is in `/$asset$/` is just library code
+      const isExternalLibraryImport = resolvedFileName.startsWith(ASSETS)
+        ? true
+        : false;
```

Comment:
These two lines seem redundant

Reply:
Prettier reformatted them this way... 🤷‍♂️

Reply:
But of course `startsWith` returns `true` or `false`... _sigh_ silly me.